website/source/docs/agent/config_entries.html.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: "docs"
+page_title: "Configuration Entry Definitions"
+sidebar_current: "docs-agent-config-entries"
+description: |-
+  Consul allows storing configuration entries centrally to be used as defaults for configuring other aspects of Consul.
+---
+
+# Configuration Entries
+
+Configuration entries can be created to provide cluster-wide defaults for various aspects of Consul. Every configuration
+entry has at least two fields: `Kind` and `Name`. Those two fields are used to uniquely identify a configuration entry.
+When put into configuration files, configuration entries can be specified as HCL or JSON objects.
+
+Example:
+
+```hcl
+Kind = "<supported kind>"
+Name = "<name of entry>"
+```
+
+Other `Kind` specific fields will be detailed below.
+
+## Usage
+
+When the agent is [configured](/docs/agent/options.html#enable_central_service_config) to enable central service configurations,
+it will look for service configuration defaults that match a registering service instance. If it finds any, the agent will merge
+those defaults with the service instance configuration. This allows for things like service protocol or proxy configuration to
+be defined globally and inherited by any affected service registrations.
+
+## Configuration Entry Kinds
+
+### Proxy Defaults - `proxy-defaults`
+
+Proxy defaults allow for configuring global config defaults across all services for Connect proxy configuration. Currently,
+only one global entry is supported.
+
+```hcl
+Kind = "proxy-defaults"
+Name = "global"
+Config {
+   proxy_specific_value = "foo"
+}
+```
+
+* `Kind` - Must be set to `proxy-defaults`
+
+* `Name` - Must be set to `global`
+
+* `Config` - An arbitrary map of configuration values used by Connect proxies.
+
+### Service Defaults - `service-defaults`
+
+Service defaults control default values for a service specified by its name, such as its protocol.
+
+```hcl
+Kind = "service-defaults"
+Name = "web"
+Protocol = "http"
+```
+
+* `Kind` - Must be set to `service-defaults`
+
+* `Name` - Set to the name of the service being configured.
+
+* `Protocol` - Sets the protocol of the service. This is used by Connect proxies for observability features.
+
+## Bootstrapping
+
+Configuration entries can be bootstrapped by putting them into the Consul server's [configuration file](/docs/agent/options.html#config_entries_bootstrap).
+Each entry is embedded inline within the configuration and when that server gains leadership it will attempt to initialize that
```

Comment:
This sentence is a little hard to read. I believe we (Judith and Kyle) updated a similar sentence in one of Kyle's PRs. I can help reword this later if the other PR language doesn't work.

Reply:
How does this sound? Did it lose any meaning?

"Configuration entries can be bootstrapped by adding them inline to each Consul server’s configuration file. When a server gains leadership, it will attempt to initialize the configuration entries. If a configuration entry does not already exist outside of the servers configuration, then it will create it. If a configuration entry does exist, that matches both `kind` and `name`, then the server will do nothing."

Reply:
Sounds great to me.